client/app/components/Modal/Modal.scss

```diff
@@ -22,15 +22,16 @@
   &Box {
     @include setFontSize($size-20);
     @include fadeIn(0.5s);
-    padding: $size-20;
+    padding: $size-30 $size-50;
+    position: relative;
     font-weight: $font-weight-400;
     color: $black;
     box-sizing: border-box;
     box-shadow: $size-0 $size-2 $size-10 $black-10;
     border-radius: $size-4;
     background: $white;
-    width: 50vw;
-    height: 60vh;
+    width: 60vw;
+    height: 65vh;
```

Comment:
Also a great start. I think it would be great to have the modal container expand to fit all the content, and then let the user scroll to see more if it overflows past the visible viewport.

Reply:
100% agree! A bunch of this stuff was pre-existing and I'm super gun-shy when making modifications to existing code on a project I don't own. Maybe I should trust my own gut a little more 👍

Reply:
To expand on these comments and the ones above here, I'm thinking that we should have a max-height and max-width set + overflow: auto. Give the modal a little room to breathe responsively but also not get disproportionately long height-wise before hitting the final breakpoint and going fullscreen.

Reply:
I messed around with the idea you presented–allowing the modal height to expand based on the content height and then scrolling the window over the overlay. It presents a host of new issues to correct unfortunately as flexbox positions the element in the center of the window right now based on the center point of the element, and not based on the top-left corner. 

So, essentially, when the content-height expands beyond the height of the screen, the top of the modal disappears and is unreachable. We can correct and reposition it, but we'd have to do that based on content-height and using JS, I think. Maybe @julianguyen has another solution! I've left it intact right now, giving it a max-height to match the Figma comp (as well as the max-width previously discussed above).

Reply:
I messed around with the idea you presented–allowing the modal height to expand based on the content height and then scrolling the window over the overlay. It presents a host of new issues to correct unfortunately as flexbox positions the element in the center of the window right now based on the center point of the element, and not based on the top-left corner. 

So, essentially, when the content-height expands beyond the height of the screen, the top of the modal disappears and is unreachable. We can correct and reposition it, but we'd have to do that based on content-height and using JS, I think. Maybe @julianguyen has another solution! I've left it intact right now, giving it a max-height to match the Figma comp (as well as the max-width previously discussed above).